### ISSUE TITLE
a11y: Add ignore rules for failing test

### ISSUE THREAD

**Issue by tlabaj** [patternfly/patternfly-react#10489]
**Please describe the issue**
 Add ignore rules for failing test.  These test pass in min branch so the are preexisting issues.

**Comment by tlabaj** [comment]
closed by #10487
